set-api-deployment-hash/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Set deployment hash
+description: Write SHA for current deployment to S3
+
+inputs:
+  s3-bucket:
+    description: S3 bucket
+    required: true
+  deployment-reference-file:
+    description: Filename to write the hash to and upload to S3
+    required: false
+    default: 'active-deployment-sha-${{ github.event.deployment.environment }}.txt'
+
+runs:
+  using: composite
+  steps:
+    - name: Update active deployment sha
+      run: |
+        echo -e "${{ github.sha }}" > ${{ inputs.deployment-reference-file }}
```

Comment:
How do you feel about making "deployment-hash" an input as well? I understand that the current version works and have also check that `${{ github.sha }}` is resolved correctly, but I'm thinking that having this as an input would make it more explicit and easier to verify that the correct hash is used.

```suggestion
        echo -e "${{ inputs.deploylment-hash }}" > ${{ inputs.deployment-reference-file }}
```

Reply:
As discussed with @felix11h we don't need it as an input. There is no use-case so far to override the values.
We've decided to use the github context, so `GITHUB_REPOSITORY` will be updated to `${{ github.repository }}`, same for `GITHUB_REPOSITORY_OWNER`.